app/enterprise/1.3-x/developer-portal/structure-and-file-types.md

```diff
@@ -147,12 +189,11 @@ x-headmatter
 ...
 ```
 
-Spec file contents themselves should be valid OAS or Swagger specifications.  If you would like to inject headmatter to the specification, you can do so by including an `x-headmatter` key to the root of the spec object.  This may be useful if you wanted to for example provide your own renderer template via `headmatter.layout` or override the specs default route via `headmatter.route`. 
+Spec file contents themselves should be valid OAS or Swagger specifications.  If you would like to inject headmatter into the specification, you can do so by including an `x-headmatter` key to the root of the spec object.  This may be useful if you wanted to for example provide your own renderer template via `headmatter.layout` or override the specs default route via `headmatter.route`. 
```

Comment:
```suggestion
Spec file contents themselves should be valid OAS or Swagger specifications.  If you would like to inject headmatter into the specification, you can do so by including an `x-headmatter` key to the root of the spec object.  This may be useful if you wanted to for example provide your own renderer template via `x-headmatter.layout` or override the specs default route via `x-headmatter.route`. 
```